SWE4040Project/SchedulerSWE4040
# Create new employee

Create a new employee with name and password
-hardcoded: company - 3
-hardcoded: location - 2

Test:
1. login screen -> enter username and password -> press create new user
== New Employee Created

2. login screen -> enter existing employee username
== See error saying duplicate employee

3. login screen -> enter username and password -> press create new user -> then press login with the same credentials
== login success
